Flaky test: should multiselect two thoughts at once

`[src/e2e/puppeteer/__tests__/multiselect.ts > mobile only > should multiselect two thoughts at once: src/e2e/puppeteer/__tests__/multiselect.ts#L49](https://github.com/cybersemics/em/pull/3334/files#annotation_41078914323)
AssertionError: expected 1 to be 2 // Object.is equality`

https://github.com/cybersemics/em/actions/runs/19037285983/attempts/1

--- a/src/e2e/puppeteer/__tests__/multiselect.ts
+++ b/src/e2e/puppeteer/__tests__/multiselect.ts
@@ -43,7 +43,8 @@ describe('mobile only', () => {
     await longPressThought(a, { edge: 'right', x: 100 })
     await longPressThought(b, { edge: 'right', x: 100 })
 
-    // Wait for the command menu panel to show "2 thoughts selected before we query for highlighted bullets, preventing race conditions in CI
+    // In CI, sometimes the count of highlighted bullets are incorrect. The selector query runs immediately after both long presses, but react might not have finished re-rendering all bullet components.
+    // Wait for the command menu panel to show "2 thoughts selected" before we query for highlighted bullets.
     await page.waitForFunction(
       () => {
         const panel = document.querySelector('[data-testid=command-menu-panel]')

